### DISCUSSION TITLE
Slint 1.0: The Next-Generation Native GUI Toolkit Matures

### DISCUSSION THREAD

**Discussion by ogoffart** [slint-ui/slint#2465] [Blog Comments]
Discussion for the [Slint 1.0 release announcement blog](https://slint-ui.com/blog/announcing-slint-1.0.html)

**Comment by suy** [comment]
Congratulations on the release! Do you know if there is some C++/JS application using Slint out there? I mean, not just a demo or example, but some 3rd party project, like the ones mentioned in Made with Slint.

**Reply by hunger** [reply]
There is proprietary C++ code close to release, but that will probably not help you too much :-/ I am not aware of any proper JS applications using Slint at this time.

Please speak up if you have some cool open source projects using Slint (in any language)!
